CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+* Add antipatterns to retry strategy by [@peter-csala](https://github.com/peter-csala) in https://github.com/App-vNext/Polly/pull/1603
+* [Docs] Reactive strategies extensibility by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1606
+* Add antipatterns to fallback strategy by [@peter-csala](https://github.com/peter-csala) in https://github.com/App-vNext/Polly/pull/1607
+* Avoid capturing where possible. by [@IEvangelist](https://github.com/IEvangelist) in https://github.com/App-vNext/Polly/pull/1609
+* Fix urls within readme extensions project by [@wahid-moh](https://github.com/wahid-moh) in https://github.com/App-vNext/Polly/pull/1616
+* [Docs] Testing by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1608
+* [Docs] Add antipatterns to circuit breaker documentation page by [@peter-csala](https://github.com/peter-csala) in https://github.com/App-vNext/Polly/pull/1621
+* [Docs] Performance docs improvements by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1618
+* [Docs] Polish the docs by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1619
+* Introduce `RetryStrategyOptions.MaxDelay` property by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1620
+* Use SHA256 for ChecksumAlgorithm and fix Binskim warning by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1623
+* [Docs] Add an antipattern to the DI documentation page by [@peter-csala](https://github.com/peter-csala) in https://github.com/App-vNext/Polly/pull/1624
+* Harden gh-pages workflow by [@martincostello](https://github.com/martincostello) in https://github.com/App-vNext/Polly/pull/1628
+* Fix typos by [@martincostello](https://github.com/martincostello) in https://github.com/App-vNext/Polly/pull/1627
+* Update README by [@martincostello](https://github.com/martincostello) in https://github.com/App-vNext/Polly/pull/1626
+* [Docs] Timeout policy migration by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1622
+* Fix the behavior of `OnHedging` event by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1625
+* Fix typo by [@martincostello](https://github.com/martincostello) in https://github.com/App-vNext/Polly/pull/1629
+* [Docs] Update some README links to point to pollydocs.org by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1634
+* [Docs] Document Retry's `MaxDelay` property by [@martintmk](https://github.com/martintmk) in https://github.com/App-vNext/Polly/pull/1631
+* Bump actionlint by [@martincostello](https://github.com/martincostello) in https://github.com/App-vNext/Polly/pull/1637
+* Documentation updates by [@martincostello](https://github.com/martincostello) in https://github.com/App-vNext/Polly/pull/1630
```

Comment:
More stuff will propably come here before merging...

Reply:
Sure, but we can add them later (or not bother if it's just minor docs stuff).

Reply:
For example, I already manually removed all the entries from a bot.